XamarinCommunityToolkitSample/Resx/AppResources.resx

```diff
@@ -143,7 +142,24 @@
          <data name="NumericValidationBehaviorDescription" xml:space="preserve">
               <value>Text color will be changed according to the configured style in case of an invalid numeric format or when the numeric value is outside of the configured boundaries (min: 1.0, max: 100.0).</value>
           </data>
+
+          <!--ConfirmPassword-->
+
+          <data name="ConfirmPasswordDescription" xml:space="preserve">
+              <value>Text color will change accordingly to the style that is configured when password confirmation doesn't match password.</value>
```

Comment:
Could you move these texts to the resource file to make it suitable for translations?

Reply:
sorry I didn't get this point already adds all strings in the resource file!

Reply:
Oh wow sorry. Yes! Totally my bad, it was late when I was looking at this 😅